### ISSUE TITLE
Add run with discreteGPU option

### ISSUE THREAD

**Issue by l2nkxk8z** [Faugus/faugus-launcher#9]
Other launchers have an option for Hybrid graphics laptops to make sure to run with the discreteGPU instead of iGPU. would be nice to have that here too.

**Comment by Faugus** [comment]
@l2nkxk8z Thanks for the feedback.
I added an option to use the discrete GPU.
Could you give it a try and see if it's working properly?
I have no way to test it.
![image](https://github.com/user-attachments/assets/4c1e2c74-d2bd-455b-8be7-faf79cf7ec3c)

**Comment by l2nkxk8z** [comment]
So I tested it by running faugus with __NV_PRIME_RENDER_OFFLOAD=0 __GLX_VENDOR_LIBRARY_NAME=intel and WINED3D and then turning the option on or off; it's working. 

One more thing though:
When I opened this issue I was on Fedora and the problem I had was that the game would run in discrete, but if I put gamescope in launch arguments it would attempt to run with iGPU and fail because of some error related to iGPU, unless I ran faugus with the offload the environment variable. Since I have switched from Fedora for now I'm not able to check if this issue has been fixed; I'm not getting the error regardless of the option on my current setup, but as long as the dGPU option comes before the launch arguments I'm assuming it would be fixed.
